frontend/hooks/useTeamIdParam.ts

```diff
@@ -343,6 +343,10 @@ export const useTeamIdParam = ({
     isTeamObserver:
       !!currentTeam?.id &&
       permissions.isTeamObserver(currentUser, currentTeam.id),
+    isObserverPlus:
```

Comment:
isTeamObserverPlus would be more consistent with the other fields here

Reply:
This accounts for both global observer plus and team observer plus, should I make two variables
`isTeamObserverPlus` and `isGlobalObserverPlus`?

Reply:
Oh I see, `currentTeam.id` will be -1 for global. Isn't that also the case for every other field here?

Reply:
Okay – currentUser.teams will never contain an "all teams" team, so these are all accurate. Looks good.